readyapi4j-core/src/main/java/com/smartbear/readyapi4j/TestRecipe.java

```diff
@@ -4,18 +4,25 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.smartbear.readyapi.client.model.TestCase;
+import com.smartbear.readyapi4j.extractor.ExtractorData;
 
 public class TestRecipe {
     private final TestCase testCase;
+    private ExtractorData extractorData;
 
-    public TestRecipe(TestCase testCase) {
+    public TestRecipe(TestCase testCase, ExtractorData extractorData) {
```

Comment:
This change will break backward-compatibility. Not sure that's a big problem, but it would also be convenient to have a one parameter constructor if we can.

Is it OK to pass null here? If so I suggest making the new parameter @Nullable and introducing a one-parameter constructor that calls this one.